app/models/game.rb

```diff
@@ -1,4 +1,63 @@
 class Game < ActiveRecord::Base
   belongs_to :users
```

Comment:
If we add `game: self` to the attributes for creating pieces, we don't need to do `pieces <<`

```ruby
Rook.create(game: self, x_cord: 1 , y_cord: 1, color: 'white')
```